.github/workflows/ci.yml

```diff
@@ -18,17 +18,4 @@ jobs:
     - run: npm -v
     - run: npm ci
     - run: npm run lint
-
-  test:
-
-    runs-on: ubuntu-latest
-
-    steps:
-    - uses: actions/checkout@v3
-    - uses: volta-cli/action@v4
-    - run: npm ci
-    - run: npm link
-    - run: |
-        cd ./tests/fixtures/build-number
-        npm i
     - run: npm test
```

Comment:
I was able to simplify all this by having the test do the `npm i` itself and import in the run function directly so the `npm link` is not needed. This makes running the code locally much easier.

Reply:
This is a much cleaner approach